pkg/aws/eni/routing/routing.go

```diff
@@ -24,6 +24,7 @@ import (
 	"github.com/cilium/cilium/pkg/logging"
 	"github.com/cilium/cilium/pkg/logging/logfields"
 	"github.com/cilium/cilium/pkg/mac"
+	"github.com/cilium/cilium/pkg/option"
```

Comment:
Travis is complaining about this line as it is an unused import. Running `goimports` on the code should take care of anything :)